packages/ui-new/src/utils/ThemeUtil.ts

```diff
@@ -99,10 +165,7 @@ function createRootStyles(themeVariables?: ThemeVariables) {
             ? `${themeVariables['--w3m-color-mix-strength']}%`
             : '0%'
         )};
-        --w3m-font-family: ${unsafeCSS(
-          themeVariables?.['--w3m-font-family'] ||
-            'Inter, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;'
-        )};
+        --w3m-font-family: ${unsafeCSS(themeVariables?.['--w3m-font-family'] || 'KHTekaMono')};
```

Comment:
One thing here, IMO we might better to provide fallback fonts as system fonts, wdyt? 

```suggestion
        --w3m-font-family: ${unsafeCSS(themeVariables?.['--w3m-font-family'] || 'KHTekaMono, system-ui, -apple-system, 'Segoe UI', Roboto, Oxygen, Ubuntu, 'Open Sans', 'Helvetica Neue', sans-serif')};
```